metis-dereference/metis-dereference-rest/src/main/resources/dereferencing.properties.example

```diff
@@ -12,4 +12,6 @@ mongo.username=
 mongo.password=
 mongo.application.name=
 entity.db=
-vocabulary.db=
\ No newline at end of file
+vocabulary.db=
+
+valid.directories.values=
```

Comment:
Exactly, that was what I meant!
But isn't the naming confusing? We are passing here url prefixes and not the actual "values". I think same applies for the naming convention down the scope.